smart-docs/docs/tutorial-review-projects.rst

```diff
@@ -8,30 +8,63 @@ Projects Page
+
+
+The image below is the project page for User1. We see that the user is a Coder for all projects except for the ``Hot Dog Classifier`` project, where it is an Admin.
 
 |lotsofprojects|
 
+
+.. _projectgroups:
+
+Adding a Project to a Group
+---------------------------
+
+To add a project to a group, we must first create the group and add one project to it by clicking the "New Group" button, writing the name of the group, and adding a project.
+
+Below we add our "About Cats Classifier" project to a new group.
+
+|projects-group|
+
+
+You can add more projects to that group by clicking "New Folder" and typing the same group name,
+but you can also add projects to the list of existing groups through selecting that project in the project page and then going to ``Details``-> ``Update Project`` -> ``Add Project to Folder``.
+
+Below we add out "About Cats Labeling" project to the Group we made for the "About Cats Classifier" project.
+
+|projects-update-group|
+
 .. _details:
 
 Details Page
 ------------
 
 The Details page provides an overview of the information and settings for your project. Each project has its own Details page, which is created when you start a new project. You can navigate to any project Details page from the :ref:`Projects Page <projects>` or to a specific Details page by pressing the "Details" link on the top navigation bar when on a project :ref:`annotationpage` page or :ref:`Admin Dashboard <admindashboard>` page.
 
+(Examples in this section are from the "About Cats Classifier" project)
+
 **The Details page lets you review:**
 
 * The project Description
 
 |details-description|
 
 * What permissions have been assigned to what users
+* The advanced settings (i.e. :ref:`active-learning`, :ref:`irr`, classifier, batch size, :ref:`projectmetadata`, deduplication settings)
 
-|details-permissions|
+|details-advanced-permissions|
 
-* The advanced settings (i.e. :ref:`active-learning`, :ref:`irr`, classifier, batch size)
+* The status of the data currently loaded in your project. This splits into:
+   * *Fully Labeled*: data which has been either labeled by one user (if not IRR data) or has received the required number of labels and they either agreed or the disagreement was resolved (if IRR data).
+   * *Unlabeled and Unassigned*: This data has not been touched since it was loaded into SMART
+   * *Awaiting Adjudication*: This data is sitting in the :ref:`adminannotate` table waiting review by an administrator. It was either IRR data where the coders did not agree, or the data was sent to an administrator by a Coder.
+   * *Recycle Bin*: This data was dropped from the project for some reason and is not included in the data totals. 
+   * *Assigned/Partially Labeled IRR*: This data is in progress. Either it has been passed out to someone for coding, or it is IRR data which has received some labels but not enough to be either adjudicated or resolved.
```

Comment:
Consider adding a link to the "Unassign Data" feature in the "Assigned/Partially Labeled IRR" as people may look here to understand why they still have outstanding unlabeled data